### PR TITLE
Adds omc opinions to the style guide

### PR DESCRIPTION
**Pull request by rauhryan**
Feedback welcome, please contribute

### CODE REVIEW

**Comment by rauhryan** [comment]
ignore the wip bot on this one

File: README.md [hunk 1]
```diff
@@ -35,33 +35,28 @@ request.
 ## Branches
 
 * Choose *short* and *descriptive* names:
+* Namespace your branches with your username
```
Comment (drusellers) [inline README.md:20]: Why?

Comment (drusellers) [reply README.md:20]: Rather, i'd enjoy seeing your why in the guide.

**Comment by drusellers** [comment]
In your second commit you say "Make liberal use of the `WIP` prefix" can we provide a wip workflow as well?

File: README.md (hunk 1 above)
Comment (rauhryan) [reply README.md:20]: 1. it prevents any collisions with branches; ie we both name a branch `css-fixes` we don't collide if they are `rauhryan/css-fixes` and `drusellers/css-fixes`
2. it's a nice to have in GUI's that folder them
bonsai__branch__robsears-sm-patch-4__and_adds_omc_opinions_to_the_style_guide_by_rauhryan_ _pull_request__1_ _omc_git-style-guide

**Comment by rauhryan** [comment]
> In your second commit you say "Make liberal use of the WIP prefix" can we provide a wip workflow as well?

Yeah, I've added a `# GitHub` section which I'll expand on tomorrow

**Pull request merged**